src/components/QuizCard/QuizCard.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./QuizCard.css";
+
+const QuizCard = ({ info, questionNum, getNextQuestion, totalQuestion }) => {
+  const { question, correctAnswer, incorrectAnswers } = info;
+
+  const allOptions = [correctAnswer, ...incorrectAnswers].sort(
+    () => Math.random() - 0.5
+  );
+
+  const updateResult = (option) => {};
+
+  const submitResult = () => {};
+
+  return (
+    <div className="card">
+      <div className="card-badge">
+        <p className="card-badge-content">
+          Question {questionNum + 1}/{totalQuestion}
+        </p>
+      </div>
+      <div className="card-text">
+        <p className="card-subtitle">{question}</p>
+      </div>
+      <div className="card-action-column">
+        {allOptions.map((option, index) => (
+          <button
+            disabled={questionNum === 10}
+            key={option + Math.random()}
```

Comment:
Why math.random here? 
Suggestion: disabled could be not hardcoded

Reply:
Yup math.random can be avoided, good catch! i just added like that, will remove in final PR.